crt_portal/cts_forms/views.py

```diff
@@ -363,7 +374,7 @@ def post(self, request):
                 errors = '; '.join(profile_form.errors[key])
                 error_msg = f'Could not save profile: {errors}'
                 messages.add_message(request, messages.ERROR, error_msg)
-            return IndexView(request)
+            return redirect('/form/view')
```

Comment:
Please reverse the url lookup instead. You probably want something like `redirect(reverse('crt_forms:crt-forms-index'))`